fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -7,7 +7,8 @@ def log
 
       $stdout.sync = true
 
-      if Helper.is_test?
+      if Helper.is_test? && !ENV.key?('DEBUG')
+        $stdout.puts "Logging disabled while running tests. Force them by setting the DEBUG mode"
```

Comment:
I had used that wording because of the one I found in the doc. I changed it, and changed the doc.